sssom/constants.py

```diff
@@ -177,3 +158,19 @@ class SchemaValidationType(str, Enum):
     SchemaValidationType.JsonSchema,
     SchemaValidationType.PrefixMapCompleteness,
 ]
+
+
+class SSSOMSchemaView:
+    """
+    SchemaView class from linkml which is instantiated when necessary.
+
+    Reason for this: https://github.com/mapping-commons/sssom-py/issues/322
+    Implemented via PR: https://github.com/mapping-commons/sssom-py/pull/323
+    """
+
+    entity_reference = "EntityReference"
+    yaml = pkg_resources.resource_filename("sssom_schema", "schema/sssom_schema.yaml")
+    view = SchemaView(yaml)
```

Comment:
I am not sure, but you may have to use init() to avoid this code being run whenever the class is referenced: 

https://stackoverflow.com/questions/9056957/correct-way-to-define-class-variables-in-python

But not 100% sure.